src/pages/Home/Home.test.js

```diff
@@ -0,0 +1,338 @@
+import React from 'react';
+import 'fake-indexeddb/auto';
+
+import Home from 'pages/Home';
+import { fireEvent, render, screen, waitFor } from 'setupTests';
+
+let emptyStatistics = {
+  totalGames: 0,
+  totalWins: 0,
+  totalAttempts: [0, 0, 0, 0, 0, 0, 0],
+  currentStreak: 0,
+  lastDatePlayed: '',
+  longestStreak: 0,
+  longestStreakDate: '',
+  attemptedWords: {},
+  timeAverage: 0,
+  minTime: 0,
+  maxTime: 0,
+};
+
+jest.mock('components/common/Loading');
+jest.mock('react-confetti');
+
+jest.mock('hooks/useSlackApp', () => () => ({
+  sendMessageToChannel: jest.fn(),
+}));
+
+jest.mock('firebase/words', () => ({
+  getTodaysWord: () => ({ todaysWord: 'GENIE' }),
+}));
+
+jest.mock('firebase/dailyResults', () => ({
+  addDailyResults: () => '1234567',
+  getDailyResults: () => ({
+    docs: [],
+    error: false,
+  }),
+  updateDailyResults: jest.fn(),
+}));
+
+jest.mock('firebase/usersStatistics', () => ({
+  getUsersStatistics: () => ({
+    currentStatistics: emptyStatistics,
+  }),
+  updateUsersStatistics: () => {},
+}));
+
+const getKeyboardLetter = letter => screen.getByRole('button', { name: letter });
+
+const getLetters = buttons =>
+  buttons.filter(item => {
+    const ariaRoleDescription = item.getAttribute('aria-roledescription');
+    return ariaRoleDescription === 'letter';
+  });
+
+beforeEach(async () => {
+  render(<Home />, {
+    preloadedState: {
+      user: {
+        user: {
+          email: 'janedoe@rootstrap.com',
+          name: 'Jane Doe',
+          photo: 'https://source.unsplash.com/random/?person',
+          uid: 'ABCDEFGHIJK',
+        },
+        authenticated: true,
+      },
+    },
+  });
+
+  await waitFor(() => {
+    const emptyWord = screen.getAllByRole('group', { name: 'empty' });
+    expect(emptyWord).toHaveLength(1);
+
+    const emptyLetters = screen.getAllByRole('button', { name: 'empty' });
+    expect(emptyLetters).toHaveLength(5);
```

Comment:
WDYT about having the current word in a constant  `const TODAY_WORD = 'GENIE'` and use it here 

```suggestion
    expect(emptyLetters).toHaveLength(TODAY_WORD.length);
```